### PR TITLE
Account for left-mover in SH in SRH

### LINKED ISSUE #532: Updraft helicity and SRH in the Southern Hemisphere
Hello everybody,

Is it possible to update the post-processing codes used to calculate the storm relative helicity (SRH) and updraft helicity (UH) for the Southern Hemisphere?

/sorc/ncep_post.fd/**CALHEL.f**
/sorc/ncep_post.fd/**CALUPDHEL.f**

From what I saw in the SRH code, the left movement of the average wind is not taken into account, and in the code to calculate the UH I did not find adaptations to find the negative ascending helicity, so I should look for the minimum values of UH, in Southern Hemisphere cases.

### PR DESCRIPTION
**Pull request by hertneky**
Add code to account for the left-mover in the SH for storm-relative helicity. This was requested by gabrielcassol #532.

### CODE REVIEW

**Review by JesseMeng-NOAA — APPROVED** [review]

File: sorc/ncep_post.fd/CALHEL.f [hunk 1]
```diff
@@ -44,7 +44,7 @@ SUBROUTINE CALHEL(DEPTH,UST,VST,HELI,USHR1,VSHR1,USHR6,VSHR6)
 !
       use vrbls3d,    only: zmid, uh, vh, u, v, zint
       use vrbls2d,    only: fis, u10, v10
-      use masks,      only: lmv
+      use masks,      only: lmv, gdlat
```
Comment (WenMeng-NOAA) [inline sorc/ncep_post.fd/CALHEL.f:48]: @hertneky Can you add a change log in this routine file? Thanks!

**Comment by WenMeng-NOAA** [comment]
@hertneky Can you sync your branch with the latest commit from upstream. Your PR will be the next one for merging. Thanks!

**Review by LinZhu-NOAA — APPROVED** [review]

File: sorc/ncep_post.fd/CALHEL.f (hunk 1 above)
Comment (hertneky) [reply sorc/ncep_post.fd/CALHEL.f:48]: @WenMeng-NOAA Added thanks.

**Comment by HuiyaChuang-NOAA** [comment]
Communicated with @yangfanglin  and @BinLiu-NOAA on HAFS team regarding this helecity changes.
Fanglin didn't think GFS heleicity is widely used so had no concerns with the change.
@LinZhu-NOAA verified the HAFS output looked ok.

Also forwarded the change to @GeoffManikin-NOAA and @AliciaBentley-NOAA on MEG team so they're
aware of this change.

**Review by WenMeng-NOAA — APPROVED** [review]

**Comment by WenMeng-NOAA** [comment]
The UPP RT tests were completed on WCOSS2, Hera and Orion. The baseline for NMMB, HAFS, RAP and GFS will be recreated with this PR merged.

**Pull request merged**